executor/hash_table.go

```diff
@@ -86,9 +86,6 @@ type hashRowContainer struct {
 
 	rowContainer *chunk.RowContainer
 	memTracker   *memory.Tracker
-
```

Comment:
Why need modify this file?

Reply:
It may be copied from the old branch. I will fix it.

Reply:
Done